Apps/Scigraph/scigraph/frame.lisp

```diff
@@ -74,7 +74,7 @@ class.  So you should do the following (in the ws package, 'natch):
 	    :display-time t
 	    :text-style 
 	    (parse-text-style '(:fix :roman :normal))
-	    :initial-cursor-visibility nil
+	    ;; :initial-cursor-visibility nil
```

Comment:
please simply remove the line. since you are at this part, please also move parse-text-style to the same line as the keyword. thanks!